core/src/main/java/org/apache/carbondata/core/datastore/page/statistics/SimpleStatsResult.java

```diff
@@ -25,6 +25,8 @@
 
   Object getMax();
 
+  byte getNull();
```

Comment:
SimpleStatsResult is the statistics for a ColumnPage, what does getNull() means for a page?

Reply:
getNull() will returns Null value present in page or not.